floydker/src/floydker/build.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+from subprocess import check_call, check_output
+import os
+import logging
+import click
+import click_log
+from .utils import gen_tag_from_filepath
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument('dockerfile')
+@click_log.simple_verbosity_option()
+@click_log.init(__name__)
+def build(dockerfile):
+    dockerfile_dir = os.path.dirname(dockerfile)
+    dockerfile_name = os.path.basename(dockerfile)
+    os.chdir(dockerfile_dir)
```

Comment:
Is there any reason to chdir to this dir? docker build should work with full file path to dockerfile, no?